test/browser_tests/spec_suites/organisms/header.js

```diff
@@ -18,14 +25,19 @@ describe( 'Header', function() {
 
   beforeAll( function() {
     _dom = {
```

Comment:
Not sure. 1 test suite file to 1 organism may be easier to manage despite the redundant lookups for the menu and search when all tests are run together. It's kind of nice to see in the test suite all the dom elements that are touched, as opposed to having some of them in a shared file and some in header.js.